api/src/main/java/net/kyori/adventure/text/format/Style.java

```diff
@@ -108,7 +109,7 @@
    */
   static @NotNull Style style(final @Nullable TextColor color) {
     if (color == null) return empty();
-    return new StyleImpl(null, color, TextDecoration.State.NOT_SET, TextDecoration.State.NOT_SET, TextDecoration.State.NOT_SET, TextDecoration.State.NOT_SET, TextDecoration.State.NOT_SET, null, null, null);
+    return new StyleImpl(null, color, Stream.of(TextDecoration.values()).map(decoration -> TextDecoration.State.NOT_SET).toArray(TextDecoration.State[]::new), null, null, null);
```

Comment:
this is a joke, right?

Reply:
![](https://media2.giphy.com/media/cdkQ2bLcwOE1cEzEeL/giphy-downsized-medium.gif)